src/PKSim.Core/Snapshots/Mappers/IdentificationParameterMapper.cs

```diff
@@ -54,6 +54,12 @@ public override async Task<ModelIdentificationParameter> MapToModel(SnapshotIden
          var parameterSelections = snapshot.LinkedParameters.Select(x => parameterSelectionFrom(x, context.Project));
 
          var identificationParameter = _identificationParameterFactory.CreateFor(parameterSelections, context.ParameterIdentification);
+         if (identificationParameter == null)
```

Comment:
@Yuri05 Really this time. It was crashign in core but I forgot to actually do sthg with the new result :)